integ-test/src/test/java/org/opensearch/sql/ppl/AddColTotalsCommandIT.java

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.ppl;
+
+import static org.opensearch.sql.legacy.TestsConstants.TEST_INDEX_ACCOUNT;
+import static org.opensearch.sql.util.MatcherUtils.schema;
+import static org.opensearch.sql.util.MatcherUtils.verifySchema;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+public class AddColTotalsCommandIT extends PPLIntegTestCase {
```

Comment:
please add `AddColTotalsCommandIT` and `AddTotalsCommandIT` to `CalciteNoPushdownIT`